src/TimePicker/TimePicker.js

```diff
@@ -75,6 +76,23 @@ class TimePicker extends Component {
         }
         return value;
     };
+    getFormattedTime = value => {
+        let time = {};
+        let timeArray = value.split(':');
+        if (typeof timeArray[0] !== 'undefined' && this.props.showHour) {
+            time.hour = timeArray[0];
+        }
+        if (typeof timeArray[1] !== 'undefined' && this.props.showMinute) {
+            time.minute = timeArray[1];
+        }
+        if (typeof timeArray[2] !== 'undefined' && this.props.showSecond) {
+            time.second = timeArray[2].match(/\d+/)[0];
+            if (this.props.format12Hours) {
+                time.meridiem = timeArray[2].indexOf(this.props.localizedText.meridiemAM) !== -1 ? 0 : 1;
+            }
+        }
+        return time;
+    }
     // formatPlaceHolder = () => {
     //   let value = '';
     //   if (this.state.showHour) {
```

Comment:
Do you mind deleting this commented out code while you're working on this - it shouldn't be there. Thank you!

Reply:
Its deleted